lib/bemhtml/index.js

```diff
@@ -213,7 +213,8 @@ BEMHTML.prototype.renderMix = function renderMix(entity,
     if (typeof item === 'string')
       item = { block: item, elem: undefined };
 
-    var hasItem = item.block || item.elem;
+    var hasItem = (item.block || item.elem) &&
+      !(item.mods && item.block === entity.block);
```

Comment:
@zxqfox it’s defferent cases: 

1. elem with mixed the same elem
2. elem with mixed elem with elemMods
3. block with mixed the same block
4. block with mixed block with mods

Now 1, 2, 3 will render as duplicates but who cares? No degradation with current version, don’t afraid.

This PR about number 4 only. The reason is “I want to mix block (for CSS purposes) with mods without apply templates”.

//cc @tadatuta